manager/state/store/services.go

```diff
@@ -26,6 +27,11 @@ func init() {
 					Unique:  true,
 					Indexer: serviceIndexerByName{},
 				},
+				indexRuntime: {
+					Name:         indexRuntime,
+					AllowMissing: true,
+					Indexer:      serviceIndexerByRuntime{},
+				},
```

Comment:
Just curious - why is it necessary to index by runtime? I don't think this was part of the original plugin runtime PR.